<!DOCTYPE html>
<!-- This site was created with Hugo Blox. https://hugoblox.com -->
<!-- Last Published: June 13, 2025 --><html lang="en-us" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Hugo Blox Builder 5.9.7" />
  

  
  












  
  










  







  
  
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
  

  
  
  
    
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media="print" onload="this.media='all'">
    
  

  
  

  
  

  

  <link rel="stylesheet" href="/css/vendor-bundle.min.26c458e6907dc03073573976b7f4044e.css" media="print" onload="this.media='all'">

  
  
  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1.9.4/css/academicons.min.css" integrity="sha512-IW0nhlW5MgNydsXJO40En2EoCkTTjZhI3yuODrZIc8cQ4h1XcF53PsqDHa09NqnkXuIe0Oiyyj171BqZFwISBw==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    
    
    
    
      
      
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    

    
    
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      
        
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.f6689966c0a10712f95f034011917db0.css" />

  
  
  

  
  
  
  
  
  
  
    
    
    <link rel="stylesheet" href="/css/libs/chroma/github-light.min.css" title="hl-light" media="print" onload="this.media='all'" >
    <link rel="stylesheet" href="/css/libs/chroma/dracula.min.css" title="hl-dark" media="print" onload="this.media='all'" disabled>
  

  
  



























  
  
  






  <meta name="author" content="Matthew A. Clarke" />





  

<meta name="description" content="Portfolio website of Matthew A. Clarke." />



<link rel="alternate" hreflang="en-us" href="https://mclarke1991.github.io/" />
<link rel="canonical" href="https://mclarke1991.github.io/" />



  <link rel="manifest" href="/manifest.webmanifest" />



<link rel="icon" type="image/png" href="/media/icon_hu10978985619530458252.png" />
<link rel="apple-touch-icon" type="image/png" href="/media/icon_hu10391704213710329260.png" />

<meta name="theme-color" content="#1565c0" />










  
  






<meta property="twitter:card" content="summary" />

  <meta property="twitter:site" content="@wowchemy" />
  <meta property="twitter:creator" content="@wowchemy" />
<meta property="twitter:image" content="https://mclarke1991.github.io/media/icon_hu10226378635250344412.png" />



  
    
    <meta property="profile:first_name" content="Matthew">
    <meta property="profile:last_name" content="Clarke">
  

<meta property="og:type" content="profile" />
<meta property="og:site_name" content="Matthew A. Clarke" />
<meta property="og:url" content="https://mclarke1991.github.io/" />
<meta property="og:title" content="Matthew A. Clarke" />
<meta property="og:description" content="Portfolio website of Matthew A. Clarke." /><meta property="og:image" content="https://mclarke1991.github.io/media/icon_hu10226378635250344412.png" /><meta property="og:locale" content="en-us" />

  
    <meta property="og:updated_time" content="2022-10-24T00:00:00&#43;00:00" />
  





<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "WebSite",
  "potentialAction": {
    "@type": "SearchAction",
    "target": "https://mclarke1991.github.io/?q={search_term_string}",
    "query-input": "required name=search_term_string"
  },
  "url": "https://mclarke1991.github.io/"
}
</script>


  




  
  
  

  
  
    <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Matthew A. Clarke" />
  

  


  
  <title>Matthew A. Clarke</title>

  
  
  
  











</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#navbar-main" class="page-wrapper   " data-wc-page-id="3976528693a0108357f4928017600865" >

  
  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.9e4214442a7711d35691acd58f6f6361.js"></script>

  


<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header header--fixed">
  
  
  
  
  












<header>
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/">Matthew A. Clarke</a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/">Matthew A. Clarke</a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#about" data-target="#about"><span>Home</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#projects" data-target="#projects"><span>Projects</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#talks" data-target="#talks"><span>Talks</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#featured" data-target="#featured"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#contact" data-target="#contact"><span>Contact</span></a>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
        

        
        
        
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        
        <li class="nav-item dropdown theme-dropdown">
          <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
            <i class="fas fa-moon" aria-hidden="true"></i>
          </a>
          <div class="dropdown-menu">
            <a href="#" class="dropdown-item js-set-theme-light">
              <span>Light</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-dark">
              <span>Dark</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-auto">
              <span>Automatic</span>
            </a>
          </div>
        </li>
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    
    
    

    

<span class="js-widget-page d-none"></span>



  
  
    





























































<section id="about" class="home-section wg-about-avatar  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  

    









  









<div id="profile">

  
  <div class="avatar-wrapper">
    
    <img class="avatar avatar-circle" src="/authors/admin/avatar_hu6902030030087730027.jpg" alt="Matthew A. Clarke"
         width="150" height="150">
    
  </div>
  

  <div class="portrait-title">

    <h2>Matthew A. Clarke</h2>

    <h3>Research Scientist, Associate Staff Visitor</h3>

    
    <h3>
      <a href="https://www.aisi.gov.uk/" target="_blank" rel="noopener">
      <span>AI Security Institute</span>
      </a>
    </h3>
    
    <h3>
      <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer/our-research/computational-cancer-biology" target="_blank" rel="noopener">
      <span>Cancer Institute, University College London</span>
      </a>
    </h3>
    
  </div>

  <ul class="network-icon" aria-hidden="true">
    
    
    
    
      
    
    
    
    
    
      
      
    
    <li>
      <a href="/#contact"  aria-label="envelope"
         >
        <i class="fas fa-envelope big-icon"></i>
      </a>
    </li>
    
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://scholar.google.co.uk/citations?user=zW8JavsAAAAJ&amp;hl=en" target="_blank" rel="noopener" aria-label="google-scholar"
         >
        <i class="ai ai-google-scholar big-icon"></i>
      </a>
    </li>
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://github.com/MClarke1991" target="_blank" rel="noopener" aria-label="github"
         >
        <i class="fab fa-github big-icon"></i>
      </a>
    </li>
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.linkedin.com/in/matthew-alan-clarke/" target="_blank" rel="noopener" aria-label="linkedin"
         >
        <i class="fab fa-linkedin big-icon"></i>
      </a>
    </li>
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.lesswrong.com/users/matthew-a-clarke" target="_blank" rel="noopener" aria-label="compass"
         >
        <i class="fas fa-compass big-icon"></i>
      </a>
    </li>
    
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://orcid.org/0000-0002-2582-9689" target="_blank" rel="noopener" aria-label="orcid"
         >
        <i class="ai ai-orcid big-icon"></i>
      </a>
    </li>
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://bsky.app/profile/maclarke.bsky.social" target="_blank" rel="noopener" aria-label="message"
         >
        <i class="fas fa-message big-icon"></i>
      </a>
    </li>
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://share.streamlit.io/user/mclarke1991" target="_blank" rel="noopener" aria-label="database"
         >
        <i class="fas fa-database big-icon"></i>
      </a>
    </li>
    
  </ul>

  
  <div class="article-style pt-2 d-flex justify-content-center">
    <div class="bio-text">
      <!-- I am driven by the desire to make a positive impact in the fields of cancer research and computational biology. I focus on how computational modelling can help us predict cancer evolution, so we can plan treatment programmes to avert or overcome resistance.  -->
<!-- {style="text-align: justify;"} -->
<!-- My passion for science began during my undergraduate studies in natural sciences at the University of Cambridge, where I developed a strong foundation in the principles of biology, physics and mathematics, and a fascination with the interface between them. I went on to earn my PhD in computational biology at Cambridge, where I refined my skills in developing complex models and algorithms. This background allows me to work at the exciting interface between biology and computer science.  -->
<!-- {style="text-align: justify;"} -->
<!-- It is also very important to me to share my knowledge and expertise with others. As a mentor to postdocs, PhD students, Masters students, and undergraduates, I take great pride in helping to guide and inspire the next generation of scientists. -->
<!-- {style="text-align: justify;"} -->
<p style="text-align: justify;">My current research focuses on understanding and predicting the safety and security of AI systems, applying my prior experience in mechanistic interpretability of both biological and computational networks.</p>
<p style="text-align: justify;">I was previously a postdoc working at the Cancer Institute at <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer" target="_blank" rel="noopener">University College London</a>. My main research interests combine biology and computer science, using computational modelling to predict cancer evolution and plan treatment programmes to avert or overcome resistance.</p>
<p style="text-align: justify;">I completed my PhD at the the <a href="https://www.cam.ac.uk/" target="_blank" rel="noopener">University of Cambridge</a>, looking at how computational network models could be used to find <a href="https://www.pnas.org/doi/abs/10.1073/pnas.1903485116" target="_blank" rel="noopener">more effective combination treatments for breast cancer</a>. As a postdoc at the <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer/our-research/computational-cancer-biology" target="_blank" rel="noopener">Fisher Lab</a> in the <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer" target="_blank" rel="noopener">UCL Cancer Institute</a> I built upon this work in order to predict resistance mechanisms to <a href="https://crukcambridgecentre.org.uk/research/strategic/radnet" target="_blank" rel="noopener">radiotherapy</a> and to find the most effective patient-specific treatments to overcome them.</p>
<!-- I am keen to share my knowledge and expertise with others. As a mentor to postdocs, PhD students, Masters students, and undergraduates, I take pride in helping to guide and inspire the next generation of scientists. If you have any questions about my work, please get in touch with the contact form below. 
{style="text-align: justify;"} -->

    </div>
  </div>
  

</div>


  

  </div>
</section>

  

  
  
    





























































<section id="section-experience" class="home-section wg-experience  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  ">
    
      
        <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Experience</h1>
          
        </div>
      
    
  

    









<div class="col-12 col-lg-8">
  

  
  

  
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col ">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border exp-fill">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.aisi.gov.uk/" target="_blank" rel="noopener"><img src="/media/icons/brands/aisi_logo.svg" width="56" height="56" alt="AI Security Institute" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Research Scientist</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.aisi.gov.uk/" target="_blank" rel="noopener">AI Security Institute</a></div>
              <div class="text-muted exp-meta">
                June 2025 –
                
                  Present
                
                
                  <span class="middot-divider"></span>
                  <span>London, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text">I am working on understanding the safety and security of frontier AI systems. I am applying my prior experience in mechanistic interpretability of both computational and biological systems; how to understand, predict and evaluate their behaviour.</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border exp-fill">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.ucl.ac.uk/cancer/" target="_blank" rel="noopener"><img src="/media/icons/brands/ucl_ci_logo.svg" width="56" height="56" alt="Cancer Institute, University College London" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Associate Staff Visitor</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.ucl.ac.uk/cancer/" target="_blank" rel="noopener">Cancer Institute, University College London</a></div>
              <div class="text-muted exp-meta">
                June 2024 –
                
                  Present
                
                
                  <span class="middot-divider"></span>
                  <span>London, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>I am continuing my work on triple-negative breast cancer, <a href="https://www.biorxiv.org/content/10.1101/2025.01.07.631497v1" target="_blank" rel="noopener">non-small cell lung cancer</a> and the automation of network generation using message-passing graph neural networks in the <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer/our-research/computational-cancer-biology" target="_blank" rel="noopener">Jasmin Fisher lab</a>.</p>
<p>Responsibilities included:</p>
<ul>
<li>Building network models by collating data on biochemical interactions in collaboration with experimental partners.</li>
<li>Development of new methods for the generation and analysis of executable network models.</li>
<li>Developing and maintaining research software developed and used by the lab.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://sparai.org/" target="_blank" rel="noopener"><img src="/media/icons/brands/spar_logo.svg" width="56" height="56" alt="Supervised Program for Alignment Research (SPAR)" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Research Fellow</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://sparai.org/" target="_blank" rel="noopener">Supervised Program for Alignment Research (SPAR)</a></div>
              <div class="text-muted exp-meta">
                February 2025 –
                
                  June 2025
                
                
                  <span class="middot-divider"></span>
                  <span>London, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>My research focused on understanding the potential for safer AI systems using gradient routing <a href="https://arxiv.org/abs/2410.04332" target="_blank" rel="noopener">Cloud et al., 2024</a>. I was mentored by <a href="https://www.linkedin.com/in/kacloud" target="_blank" rel="noopener">Alex Cloud</a>.</p>
<p>Responsibilities included:</p>
<ul>
<li>Organising and planning research for the team.</li>
<li>Planning and running experiments with frontier LLMs including GPT-4o and Qwen3.</li>
<li>Writing research reports and presentations.</li>
<li>Presenting at conferences.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://pibbss.ai/" target="_blank" rel="noopener"><img src="/media/icons/brands/pibbss_logo.svg" width="56" height="56" alt="Principles of Intelligent Behavior in Biological and Social Systems (PIBBSS)" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Research Fellow</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://pibbss.ai/" target="_blank" rel="noopener">Principles of Intelligent Behavior in Biological and Social Systems (PIBBSS)</a></div>
              <div class="text-muted exp-meta">
                June 2024 –
                
                  December 2024
                
                
                  <span class="middot-divider"></span>
                  <span>London, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>My research focused on the mechanistic interpretability of machine learning, specifically using sparse autoencoders (SAEs). This work combined my interests in computational modeling and complex systems, now applied to understanding the inner workings of AI. I was mentored by <a href="https://www.linkedin.com/in/joseph-bloom1/" target="_blank" rel="noopener">Joseph Bloom</a>. We found that SAE latents are not always independent, <a href="https://www.lesswrong.com/posts/WNoqEivcCSg8gJe5h/compositionality-and-ambiguity-latent-co-occurrence-and" target="_blank" rel="noopener">forming clusters mapping interpretable subspaces</a>, for example <a href="https://feature-cooccurrence.streamlit.app/?model=gemma-2-2b&amp;sae_release=gemma-scope-2b-pt-res-canonical&amp;sae_id=layer_12_width_16k_canonical&amp;size=4&amp;subgraph=4740" target="_blank" rel="noopener">different mixtures of latents encode different quantities in an interpretable subspace</a>.</p>
<p>Responsibilities included:</p>
<ul>
<li>Planning experiments using frontier models and LLMs.</li>
<li>Development of new methods for the analysis of SAE latents.</li>
<li>Communication of our findings through presentations and publications.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.ucl.ac.uk/cancer/" target="_blank" rel="noopener"><img src="/media/icons/brands/ucl_ci_logo.svg" width="56" height="56" alt="Cancer Institute, University College London" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Postdoctoral Research Fellow</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.ucl.ac.uk/cancer/" target="_blank" rel="noopener">Cancer Institute, University College London</a></div>
              <div class="text-muted exp-meta">
                April 2020 –
                
                  June 2024
                
                
                  <span class="middot-divider"></span>
                  <span>London, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>As part of the <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer/our-research/computational-cancer-biology" target="_blank" rel="noopener">Jasmin Fisher lab</a>, I continued my work on DNA damage repair, applying this to breast cancer and <a href="https://www.biorxiv.org/content/10.1101/2025.01.07.631497v1" target="_blank" rel="noopener">lung cancer</a>. I also developed new methods for studying cancer evolution. During the pandemic, my colleagues and I demonstrated <a href="https://www.nature.com/articles/s41746-022-00561-5.pdf" target="_blank" rel="noopener">how our work on cancer could be applied to infectious disease</a>, to predict drug repurposing for rapid response to the COVID-19 epidemic.</p>
<p>Responsibilities included:</p>
<ul>
<li>Building network models by collating data on biochemical interactions in collaboration with experimental partners.</li>
<li>Development of new methods for the generation and analysis of executable network models.</li>
<li>Supervision of undergraduate and Master&rsquo;s students.</li>
<li>Interviewing hiring candidates.</li>
<li>Mentoring new members of the lab.</li>
<li>Presenting at international conferences.</li>
<li>Developing and maintaining research software developed and used by the lab.</li>
<li>Lab Management.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.pro-matix.com/" target="_blank" rel="noopener"><img src="/media/icons/brands/promatix_logo.svg" width="56" height="56" alt="Promatix Biosciences Ltd" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Consultant</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.pro-matix.com/" target="_blank" rel="noopener">Promatix Biosciences Ltd</a></div>
              <div class="text-muted exp-meta">
                October 2021 –
                
                  September 2023
                
                
                  <span class="middot-divider"></span>
                  <span>London, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>I worked to support Promatix Ltd to <a href="https://www.ucl.ac.uk/medical-sciences/news/2022/nov/ucl-fisher-lab-partners-promatix-accelerate-hunt-oncology-therapeutics" target="_blank" rel="noopener">accelerate the hunt for oncology therapeutics</a> as part of their collaboration with the <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer/our-research/computational-cancer-biology" target="_blank" rel="noopener">Jasmin Fisher lab</a>.</p>
<p>Responsibilities included:</p>
<ul>
<li>Interviewing hiring candidates.</li>
<li>Advising on research.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener"><img src="/media/icons/brands/biochem_cam_logo.svg" width="56" height="56" alt="Department of Biochemistry, University of Cambridge" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Postdoctoral Research Associate</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener">Department of Biochemistry, University of Cambridge</a></div>
              <div class="text-muted exp-meta">
                October 2018 –
                
                  April 2020
                
                
                  <span class="middot-divider"></span>
                  <span>Cambridge, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>With funding from <a href="https://crukcambridgecentre.org.uk/research/strategic/radnet" target="_blank" rel="noopener">CRUK RadNet</a>, I worked in the <a href="https://www.ucl.ac.uk/medical-sciences/divisions/cancer/our-research/computational-cancer-biology" target="_blank" rel="noopener">Jasmin Fisher lab</a> on modelling the DNA damage response pathway to find radiosensitising drug treatments. I also continued my work on breast cancer, focussing on the triple-negative sub-type (TNBC) as part of a collaboration with the <a href="https://themarkfoundation.org/" target="_blank" rel="noopener">Mark Foundation for Cancer Research</a> and <a href="https://www.cancerresearchuk.org/about-cancer/find-a-clinical-trial/a-trial-looking-at-olaparib-with-chemotherapy-for-early-breast-cancer-partner" target="_blank" rel="noopener">PARTNER clinical trial</a>.  In collaboration with Jasmin Fisher I published a review on the <a href="https://www.nature.com/articles/s41568-020-0258-x.pdf" target="_blank" rel="noopener">opportunities and challenges for executable modelling</a>.</p>
<p>Responsibilities included:</p>
<ul>
<li>Building network models by collating data on biochemical interactions in collaboration with experimental partners.</li>
<li>Supervision of undergraduate and Master&rsquo;s students.</li>
<li>Presenting at international conferences.</li>
<li>Mentoring new members of the lab.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener"><img src="/media/icons/brands/biochem_cam_logo.svg" width="56" height="56" alt="Department of Biochemistry, University of Cambridge" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Visiting Scholar</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener">Department of Biochemistry, University of Cambridge</a></div>
              <div class="text-muted exp-meta">
                January 2018 –
                
                  October 2018
                
                
                  <span class="middot-divider"></span>
                  <span>Cambridge, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>Thanks to generous funding from the <a href="https://register-of-charities.charitycommission.gov.uk/charity-details/?regid=1110119&amp;subid=0" target="_blank" rel="noopener">Glover Fund</a>, I was able to build upon my work on the evolution of cancer, expanding it to <a href="https://www.nature.com/articles/s41467-022-33189-w.pdf" target="_blank" rel="noopener">understand how the behaviour of blood cancers is determined by the order in which they acquire mutations, even when their final mutational profile is identical</a> as part of a collaboration of the <a href="https://www.bioc.cam.ac.uk/research/former-faculty/jasmin-fisher" target="_blank" rel="noopener">Jasmin Fisher lab</a> with the <a href="https://www.hall-lab.com/research" target="_blank" rel="noopener">Hall</a> and <a href="https://www.york.ac.uk/biology/people/david-kent/" target="_blank" rel="noopener">Kent</a> labs.</p>
<p>Responsibilities included:</p>
<ul>
<li>Building network models by collating data on biochemical interactions in collaboration with experimental partners.</li>
<li>Supervision of undergraduate and Master&rsquo;s students.</li>
<li>Presenting at international conferences.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener"><img src="/media/icons/brands/biochem_cam_logo.svg" width="56" height="56" alt="Department of Biochemistry, University of Cambridge" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">PhD in Computational Biology</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener">Department of Biochemistry, University of Cambridge</a></div>
              <div class="text-muted exp-meta">
                October 2013 –
                
                  January 2018
                
                
                  <span class="middot-divider"></span>
                  <span>Cambridge, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>As part of the <a href="https://wellcome.org/" target="_blank" rel="noopener">Wellcome Trust</a> Mathematical Genomics and Medicine PhD programme, studied as part of the University of Cambridge Computational Biology MPhil, while also undertaking rotations in different labs. For my PhD I was supervised by <a href="https://www.bioc.cam.ac.uk/research/former-faculty/jasmin-fisher" target="_blank" rel="noopener">Jasmin Fisher</a> in the <a href="https://www.bioc.cam.ac.uk/" target="_blank" rel="noopener">University of Cambridge Department of Biochemistry</a> and <a href="https://www.microsoft.com/en-us/research/lab/microsoft-research-cambridge/" target="_blank" rel="noopener">Microsoft Research Cambridge</a>, and my advisor was <a href="https://www.bioc.cam.ac.uk/research/littlewood" target="_blank" rel="noopener">Trevor Littlewood</a>.  During my PhD, I developed a network model of breast cancer, as well as novel methods to study combination treatment and the <a href="https://link.springer.com/chapter/10.1007/978-3-030-17297-8_5" target="_blank" rel="noopener">evolution of cancers</a>. Using these, in collaboration with the <a href="https://www.bioc.cam.ac.uk/research/evan" target="_blank" rel="noopener">Gerard Evan lab</a>, we were able to identify and validate a <a href="https://www.pnas.org/doi/abs/10.1073/pnas.1903485116" target="_blank" rel="noopener">novel combination treatment for breast cancer</a>, exploiting the heterogeneity observed in myc driven breast cancers. My PhD examiners were <a href="https://www.stemcells.cam.ac.uk/people/pi/gottgens" target="_blank" rel="noopener">Bertie Göttgens</a> and <a href="https://www.oncology.ox.ac.uk/team/francesca-buffa" target="_blank" rel="noopener">Francesca Buffa</a>.</p>
<p>Responsibilities included:</p>
<ul>
<li>Building network models by collating data on biochemical interactions in collaboration with experimental partners.</li>
<li>Supervision of undergraduate and Master&rsquo;s students.</li>
<li>Presenting at international conferences.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.phy.cam.ac.uk/" target="_blank" rel="noopener"><img src="/media/icons/brands/cavendish_logo.svg" width="56" height="56" alt="Department of Physics, University of Cambridge" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Masters in Experimental and Theoretical Physics</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.phy.cam.ac.uk/" target="_blank" rel="noopener">Department of Physics, University of Cambridge</a></div>
              <div class="text-muted exp-meta">
                October 2012 –
                
                  October 2013
                
                
                  <span class="middot-divider"></span>
                  <span>Cambridge, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text">For my Master&rsquo;s thesis, I studyed the how fibroblasts sense and respond to mechanical changes in their environment, in particular in wound healing.  In the <a href="https://www.phy.cam.ac.uk/directory/terentjeve" target="_blank" rel="noopener">Terentjev lab</a> worked on stability analysis of non-linear differential equations modelling this system, particularly focussing on <a href="https://en.wikipedia.org/wiki/Lyapunov_stability" target="_blank" rel="noopener">Lyapunov Methods</a>.</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col ">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.cam.ac.uk/" target="_blank" rel="noopener"><img src="/media/icons/brands/cam_logo.svg" width="56" height="56" alt="University of Cambridge" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">BA (MA Cantab) in Natural Sciences</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.cam.ac.uk/" target="_blank" rel="noopener">University of Cambridge</a></div>
              <div class="text-muted exp-meta">
                October 2009 –
                
                  October 2012
                
                
                  <span class="middot-divider"></span>
                  <span>Cambridge, UK</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text">Reading Natural Sciences gave me the opportunity to build skills in diverse areas of science and mathematics, ranging from particle physics to experimental biology. This gave me the curiosity about how the programming underlying cell behaviour functions, and the mathematical and computational skills to pursue those questions.</div>
        </div>
      </div>
    </div>
  </div>
  
  
</div>


  
    </div>
  

  </div>
</section>

  

  
  
    





























































<section id="projects" class="home-section wg-portfolio  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  justify-content-center">
    
      
        <div class="section-heading col-12 mb-3 text-center">
          <h1 class="mb-0">Projects</h1>
          
        </div>
      
    
  

    












<div class="col-12 ">

  

  

    

    
    
    
    
      
      
        
      
    

    <span class="d-none default-project-filter">*</span>

    
    
  

  <div class="isotope projects-container row js-layout-row project-showcase">

    
    

    
    
    
      
    
    
    
    
      
    

    
    
    
    
    
    

    
      
        
          <div class="col-12 isotope-item js-id-drug-discovery js-id-computational-modelling js-id-biomodelanalyzer js-id-drug-discovery js-id-graph-neural-networks js-id-machine-learning">
        
        





  








  






<div class="col-lg-12 mb-5 view-showcase">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <div class="section-subheading article-title mb-0 mt-0"><a href="/project/auto_drug/" >Automating Drug Discovery</a></div>

      
      <div class="article-style">
        Computational models have become essential tools for understanding signalling networks and their non-linear dynamics. However, these models are typically constructed manually using prior knowledge and can be over-reliant on study bias. Scaling up the construction and analysisof models to take advantage of increasingly abundant &lsquo;omics data can bridge these gaps by providing a comprehensive view of signalling events and how they influence cellular phenotypes.
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.biorxiv.org/content/10.1101/2025.05.16.653408v1" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Clarke, Barker & Sun et al. (2025)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/jfisher-lab/MAGELLAN" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Clarke, Barker & Sun et al. (2025)</a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      

      
        
        
        
        <a href="/project/auto_drug/" >
          <img src="/project/auto_drug/featured_hu2523131550787785656.webp" height="516" width="540"
              alt="Automating Drug Discovery" loading="lazy" style="width: 540px; height: auto; object-fit: cover;">
        </a>
      
    </div>
  </div>
</div>

      </div>
    
      
        
          <div class="col-12 isotope-item js-id-llm js-id-sae js-id-mechanistic-interpretability js-id-pibbss js-id-machine-learning js-id-ai js-id-sparse-autoencoder js-id-ai-safety">
        
        





  








  






<div class="col-lg-12 mb-5 view-showcase">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <div class="section-subheading article-title mb-0 mt-0"><a href="/project/sae_cooccurrence/" >SAE Latent Co-occurrence</a></div>

      
      <div class="article-style">
        Sparse AutoEncoder (SAE) latents show promise as a method for extracting interpretable features from large language models (LM), but their overall utility as the foundation of mechanistic understanding of LM remains unclear. Ideal features would be linear and independent, but we show that there exist SAE latents in GPT2-Small display non-independent behaviour, especially in small SAEs. We investigate: 1. What fraction of SAE latents might best be understood in groups rather than individually? 2. Do low-dimensional subspaces mapped by co-occurring groups of features ever provide a better unit of analysis (or possibly intervention) than individual SAE latents?
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.lesswrong.com/posts/WNoqEivcCSg8gJe5h/compositionality-and-ambiguity-latent-co-occurrence-and" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Clarke et al. (2024)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/MClarke1991/sae_cooccurrence" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Clarke et al. (2024)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://feature-cooccurrence.streamlit.app/" target="_blank" rel="noopener">
    <i class="fas fa-database mr-1"></i>SAE Latent Co-occurrence Explorer</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://youtu.be/OyKXjdA09fE?si=gD9IRD0ohVMEbYwr" target="_blank" rel="noopener">
    <i class="fas fa-video mr-1"></i>Presentation</a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      

      
        
        
        
        <a href="/project/sae_cooccurrence/" >
          <img src="/project/sae_cooccurrence/featured_hu10623782521660131607.webp" height="355" width="540"
              alt="SAE Latent Co-occurrence" loading="lazy" style="width: 540px; height: auto; object-fit: cover;">
        </a>
      
    </div>
  </div>
</div>

      </div>
    
      
        
          <div class="col-12 isotope-item js-id-cancer js-id-breast-cancer js-id-lung-cancer js-id-melanoma js-id-covid-19 js-id-sars-cov-2 js-id-infectious-disease js-id-computational-modelling js-id-network-modelling js-id-biomodelanalyzer js-id-combination-therapy">
        
        





  








  






<div class="col-lg-12 mb-5 view-showcase">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <div class="section-subheading article-title mb-0 mt-0"><a href="/project/combination_therapy/" >Combination therapy</a></div>

      
      <div class="article-style">
        Computatational modelling allows screening of thousands of combinations of drugs to find the most effective treatments. This allows for better protection against the emergence of resistance, as well as rapid drug repurposing e.g. to combat emerging diseases such as COVID-19.
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.biorxiv.org/content/10.1101/2025.01.07.631497v1" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Clarke, Barker & Nicholls et al. (2025)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.pnas.org/doi/pdf/10.1073/pnas.1903485116" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Kreuzaler & Clarke et al. (2019)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/MClarke1991#:~:text=HeterogeneousBreastCancer" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Kreuzaler & Clarke et al. (2019)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.nature.com/articles/s41746-022-00561-5" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Howell, Clarke & Reuschl et al. (2022)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/JFisherLab/COVID19" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Howell, Clarke & Reuschl et al. (2022)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.science.org/doi/epdf/10.1126/sciadv.add1992" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Howell & Davies et al. (2023)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/JFisherLab/Melanoma-LC" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Howell & Davies et al. (2023)</a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      

      
        
        
        
        <a href="/project/combination_therapy/" >
          <img src="/project/combination_therapy/featured_hu16079984639851024138.webp" height="339" width="540"
              alt="Combination therapy" loading="lazy" style="width: 540px; height: auto; object-fit: cover;">
        </a>
      
    </div>
  </div>
</div>

      </div>
    
      
        
          <div class="col-12 isotope-item js-id-evolution js-id-cancer js-id-tumorigenesis js-id-binary-decision-diagram js-id-biomodelanalyzer js-id-bma js-id-breast-cancer js-id-myeloproliferative-neoplasms js-id-mpns js-id-acute-myeloid-leukemia js-id-aml">
        
        





  








  






<div class="col-lg-12 mb-5 view-showcase">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <div class="section-subheading article-title mb-0 mt-0"><a href="/project/cancer_evolution/" >Predicting Cancer Evolution</a></div>

      
      <div class="article-style">
        Cancers evolve from normal cells, and continue to evolve to evade treatment. Understanding evolution from tumourigenesis will allow for earlier detection of cancer, while predicting evolution of resistance mechanisms is essential for finding treatments that are effective in the long term.
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://link.springer.com/chapter/10.1007/978-3-030-17297-8_5" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Clarke et al. (2019)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/JFisherLab/MutationOrder/" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Clarke et al. (2019)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.nature.com/articles/s41467-022-33189-w.pdf" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Talarmain et al. (2022)</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://zenodo.org/records/6913664#.Yx7YBnZBy3A" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>Talarmain et al. (2022)</a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      

      
        
        
        
        <a href="/project/cancer_evolution/" >
          <img src="/project/cancer_evolution/featured_hu2595169889723851899.webp" height="301" width="540"
              alt="Predicting Cancer Evolution" loading="lazy" style="width: 540px; height: auto; object-fit: cover;">
        </a>
      
    </div>
  </div>
</div>

      </div>
    
      
        
          <div class="col-12 isotope-item js-id-cancer js-id-radiotherapy js-id-cruk-radnet js-id-breast-cancer js-id-lung-cancer js-id-computational-modelling js-id-network-modelling js-id-biomodelanalyzer js-id-combination-therapy js-id-evolution js-id-drug-discovery js-id-dna-damage-repair js-id-ddr">
        
        





  








  






<div class="col-lg-12 mb-5 view-showcase">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <div class="section-subheading article-title mb-0 mt-0"><a href="/project/radiotherapy/" >DNA Damage Repair and Radiotherapy</a></div>

      
      <div class="article-style">
        Cancers often emerge, in part, due to deficencies in DNA damage repair, making them vulnerable to DNA damaging treatments such as radiotherapy. While radiotherapy efficacy has been increasing, this is mainly due to better targeting of tumour anatomy, but targeted therapy offers the opportunity to radiosensitise tumour cells by targeting vulnerabilities in the tumour biology. We use computational modelling to find such vulnerabilities, and explore potential resistance mechanisms.
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.biorxiv.org/content/10.1101/2025.01.07.631497v1" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Clarke, Barker & Nicholls et al. (2025)</a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      

      
        
        
        
        <a href="/project/radiotherapy/" >
          <img src="/project/radiotherapy/featured_hu13311811188174145867.webp" height="330" width="540"
              alt="DNA Damage Repair and Radiotherapy" loading="lazy" style="width: 540px; height: auto; object-fit: cover;">
        </a>
      
    </div>
  </div>
</div>

      </div>
    
      
        
          <div class="col-12 isotope-item js-id-gradient-routing js-id-neural-networks js-id-ai-safety js-id-spar">
        
        





  








  






<div class="col-lg-12 mb-5 view-showcase">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <div class="section-subheading article-title mb-0 mt-0"><a href="/project/gradient_routing/" >Gradient Routing</a></div>

      
      <div class="article-style">
        Can we control where learning happens in neural networks? Gradient routing addresses this by applying masks to limit the flow of gradients during backpropagation. By supplying different masks for different data points, the user can induce specialized subcomponents within a model. I worked on applying this method to AI safety research, focussing on addressing the problem of <a href="https://arxiv.org/abs/2502.17424" target="_blank" rel="noopener">emergent misalignment</a> during <a href="https://arxiv.org/abs/2302.00487" target="_blank" rel="noopener">continual learning</a>.
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.lesswrong.com/posts/tAnHM3L25LwuASdpF/selective-modularity-a-research-agenda" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Cloud et al., 2025</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://arxiv.org/abs/2410.04332" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Cloud et al., 2024</a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      

      
        
        
        
        <a href="/project/gradient_routing/" >
          <img src="/project/gradient_routing/featured_hu13405528009468625267.webp" height="160" width="540"
              alt="Gradient Routing" loading="lazy" style="width: 540px; height: auto; object-fit: cover;">
        </a>
      
    </div>
  </div>
</div>

      </div>
    

  </div>
</div>


  
    </div>
  

  </div>
</section>

  

  
  
    





























































<section id="featured" class="home-section wg-collection  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  ">
    
      
        <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Featured Publications</h1>
          
        </div>
      
    
  

    










  









  
  
  
  
  







  
















  




<div class="col-12 col-lg-8">

  

  
    











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span class="author-highlighted">
      Matthew A. Clarke</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author, Co-corresponding author"></i>, <span >
      Charlie George Barker</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Yuxin Sun</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Theodoros I. Roumeliotis</span>, <span >
      Jyoti S. Choudhary</span>, <span >
      Jasmin Fisher</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2025
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      bioRxiv
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    
    <a href="/publication/magellan/" >
      <div class="img-hover-zoom">
        <img src="/publication/magellan/featured_hu18147389232024291619.webp" height="455" width="808"
            class="article-banner" alt="MAGELLAN: Automated Generation of Interpretable Computational Models for Biological Reasoning" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/magellan/" >MAGELLAN: Automated Generation of Interpretable Computational Models for Biological Reasoning</a>
  </div>

  
  <a href="/publication/magellan/"  class="summary-link">
    <div class="article-style">
      <p>Computational models have become essential tools for understanding signalling networks and their non-linear dynamics. However, these models are typically constructed manually using prior knowledge and can be over-reliant on study bias. These limitations hinder their ability to make accurate predictions and incorporate new evidence. Scaling up the construction of models to take advantage of increasingly abundant &lsquo;omics data can bridge these gaps by providing a comprehensive view of signalling events and how they influence cellular phenotypes. In this study, we present MAGELLAN, a method leveraging message passing graph neural networks to build computational models directly from pathway data and discrete rules representing experimental results. We used this to construct a computational model of breast cancer signalling and re-parameterize a previously published non-small cell lung cancer (NSCLC) model, showing that MAGELLAN can predict genetic dependencies and achieve comparable model quality to expert-curated and manually trained models. Our approach enables the integration of prior knowledge networks and experimental data to build predictive models that are mechanistically interpretable. This approach simplifies model creation, making it more accessible and practical for experimentalists, and supports broader applications in drug discovery and biological research.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2025.05.16.653408v1" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/jfisher-lab/MAGELLAN" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/auto_drug/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/https://doi.org/10.1101/2025.05.16.653408" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

  
    











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span class="author-highlighted">
      Matthew A. Clarke</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Charlie George Barker</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Ashley Nicholls</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Matt P. Handler</span>, <span >
      Lisa Pickard</span>, <span >
      Amna Shah</span>, <span >
      David Walter</span>, <span >
      Etienne De Braekeleer</span>, <span >
      Udai Banerji</span>, <span >
      Jyoti Choudhary</span>, <span >
      Saif Ahmed</span>, <span >
      Ultan McDermott</span>, <span >
      Gregory J.  Hannon</span>, <span >
      Jasmin Fisher</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January, 2025
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      bioRxiv
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    
    <a href="/publication/nsclc_2025/" >
      <div class="img-hover-zoom">
        <img src="/publication/nsclc_2025/featured_hu9258572545375064603.webp" height="455" width="808"
            class="article-banner" alt="Predicting Personalised Therapeutic Combinations in Non-Small Cell Lung Cancer Using In Silico Modelling" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/nsclc_2025/" >Predicting Personalised Therapeutic Combinations in Non-Small Cell Lung Cancer Using In Silico Modelling</a>
  </div>

  
  <a href="/publication/nsclc_2025/"  class="summary-link">
    <div class="article-style">
      <p>The disease burden from non-small cell lung cancer (NSCLC) adenocarcinoma is substantial, with around a million new cases diagnosed globally each year, and a 5-year survival rate of less than 20%. A lack of therapeutic options personalized to individual patient genetics, and the targeted therapies that exist quickly succumbing to resistance, leads to high variation in survival. Patient stratification combined with greater personalisation of therapies have the potential to improve outcomes, however, the wide variation in mutations found in NSCLC adenocarcinoma patients mean that experimentally determining suitable treatment combinations is time-consuming and expensive. Here we present an in silico model encompassing tumour intrinsic key oncogenic signalling pathways, including EGFR, AKT, JAK/STAT and WNT for efficiently predicting rational drug-drug and drug-radiotherapy combination therapies in NSCLC. Using this model, we simulate diverse genetic profiles and test over 10,000 therapeutic strategies to identify optimal strategies to overcome resistance mechanisms specific to genetic profiles and p53 status. Our in silico model reproduces drug additivity experiments, predicts radio-sensitising genes validated in a CRISPR screen and identifies 53BP1 as a potential drug target that improves the therapeutic window during radiotherapy, as well as potential to use ATM inhibitors to overcome p53 loss-of-function driven radiotherapy resistance. We further use the in silico model to identify a 19-gene signature to stratify patients most likely to benefit from radiotherapy and validated this using TCGA data. These results further demonstrate the utility of in silico mechanistic modelling and present a bespoke computational resource for large-scale screening of personalised therapies applied to NSCLC.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2025.01.07.631497v1" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/radiotherapy/">
    Project
  </a>
  











  </div>
  

</div>

  
    











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Hardik Bhatnagar</span>, <span >
      Joseph Bloom</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December, 2024
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      LessWrong
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    
    <a href="/publication/sae_cooc_2024/" >
      <div class="img-hover-zoom">
        <img src="/publication/sae_cooc_2024/featured_hu8424405631975539759.webp" height="455" width="808"
            class="article-banner" alt="Compositionality and Ambiguity: Latent Co-occurrence and Interpretable Subspaces" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/sae_cooc_2024/" >Compositionality and Ambiguity: Latent Co-occurrence and Interpretable Subspaces</a>
  </div>

  
  <a href="/publication/sae_cooc_2024/"  class="summary-link">
    <div class="article-style">
      <p>Sparse AutoEncoder (SAE) latents show promise as a method for extracting interpretable features from large language models (LM), but their overall utility for mechanistic understanding of LM remains unclear. Ideal features would be linear and independent, but we show that there exist SAE latents in GPT2-Small and Gemma-2-2b that display non-independent behaviour, especially in small SAEs. Rather, latents co-occur in clusters that map out interpretable subspaces, leading us to question how independent these latents are, and how does this depend on the SAE? We find that these subspaces show latents acting compositionally, as well as being used to resolve ambiguity in language, though SAE latents remain largely independently interpretable within these contexts despite this behaviour. Latent clusters decrease in both size and prevalence as SAE width increases, suggesting this is a phenomenon of small SAEs with coarse-grained latents. Our findings suggest that better understanding of how LM process information can be achieved in some cases by understanding SAE latents as being able to form functional units that are interpreted as a whole.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.lesswrong.com/posts/WNoqEivcCSg8gJe5h/compositionality-and-ambiguity-latent-co-occurrence-and" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/MClarke1991/sae_cooccurrence" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://feature-cooccurrence.streamlit.app/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/sae_cooccurrence/">
    Project
  </a>
  







  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://youtu.be/OyKXjdA09fE?si=gD9IRD0ohVMEbYwr" target="_blank" rel="noopener">
  Video
</a>





  </div>
  

</div>

  
    











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Rowan Howell</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span class="author-highlighted">
      Matthew A. Clarke</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Ann-Kathrin Reuschl</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Tianyi Chen</span>, <span >
      Sean Abbott-Imboden</span>, <span >
      Mervyn Singer</span>, <span >
      David M. Lowe</span>, <span >
      Clare L. Bennett</span>, <span >
      Benjamin Chain</span>, <span >
      Clare Jolly</span>, <span >
      Jasmin Fisher</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      npj Digital Medicine
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    
    <a href="/publication/howell2021/" >
      <div class="img-hover-zoom">
        <img src="/publication/howell2021/featured_hu17911242796259519039.webp" height="455" width="808"
            class="article-banner" alt="Executable Network of SARS-CoV-2-Host Interaction Predicts Drug Combination Treatments" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/howell2021/" >Executable Network of SARS-CoV-2-Host Interaction Predicts Drug Combination Treatments</a>
  </div>

  
  <a href="/publication/howell2021/"  class="summary-link">
    <div class="article-style">
      <p>The COVID-19 pandemic has pushed healthcare systems globally to a breaking point. The urgent need for effective and affordable COVID-19 treatments calls for repurposing combinations of approved drugs. The challenge is to identify which combinations are likely to be most effective and at what stages of the disease. Here, we present the first disease-stage executable signalling network model of SARS-CoV-2-host interactions used to predict effective repurposed drug combinations for treating early- and late stage severe disease. Using our executable model, we performed in silico screening of 9870 pairs of 140 potential targets and have identified nine new drug combinations. Camostat and Apilimod were predicted to be the most promising combination in effectively supressing viral replication in the early stages of severe disease and were validated experimentally in human Caco-2 cells. Our study further demonstrates the power of executable mechanistic modelling to enable rapid pre-clinical evaluation of combination therapies tailored to disease progression. It also presents a novel resource and expandable model system that can respond to further needs in the pandemic.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/articles/s41746-022-00561-5.pdf" target="_blank" rel="noopener">
  PDF
</a>





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.github.com/jfisherlab/COVID19" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/combination_therapy/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41746-022-00561-5" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

  
    











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Jasmin Fisher</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Reviews Cancer
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    
    <a href="/publication/clarke2020/" >
      <div class="img-hover-zoom">
        <img src="/publication/clarke2020/featured_hu16318121864566505325.webp" height="455" width="808"
            class="article-banner" alt="Executable cancer models: successes and challenges" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/clarke2020/" >Executable cancer models: successes and challenges</a>
  </div>

  
  <a href="/publication/clarke2020/"  class="summary-link">
    <div class="article-style">
      <p>Making decisions on how best to treat cancer patients requires the integration of different data sets, including genomic profiles, tumour histopathology, radiological images, proteomic analysis and more. This wealth of biological information calls for novel strategies to integrate such information in a meaningful, predictive and experimentally verifiable way. In this Perspective we explain how executable computational models meet this need. Such models provide a means for comprehensive data integration, can be experimentally validated, are readily interpreted both biologically and clinically, and have the potential to predict effective therapies for different cancer types and subtypes. We explain what executable models are and how they can be used to represent the dynamic biological behaviours inherent in cancer, and demonstrate how such models, when coupled with automated reasoning, facilitate our understanding of the mechanisms by which oncogenic signalling pathways regulate tumours. We explore how executable models have impacted the field of cancer research and argue that extending them to represent a tumour in a specific patient (that is, an avatar) will pave the way for improved personalized treatments and precision medicine. Finally, we highlight some of the ongoing challenges in developing executable models and stress that effective cross-disciplinary efforts are key to forward progress in the field.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/articles/s41568-020-0258-x.pdf" target="_blank" rel="noopener">
  PDF
</a>















<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41568-020-0258-x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

  

  
  
  

    
    
      
    

    
    
    
    
      
    

    

    <div class="see-all">
      <a href="/publication/">
        See all publications
        <i class="fas fa-angle-right"></i>
      </a>
    </div>
  

</div>


  
    </div>
  

  </div>
</section>

  

  
  
    





























































<section id="section-collection" class="home-section wg-collection  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  ">
    
      
        <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Recent Publications</h1>
          
        </div>
      
    
  

    










  









  
  
  
  
  






















  




<div class="col-12 col-lg-8">

  <div class="alert alert-note">
  <div>
    Quickly discover relevant content by <a href="./publication/">filtering publications</a>.
  </div>
</div>


  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/publication/magellan/" >MAGELLAN: Automated Generation of Interpretable Computational Models for Biological Reasoning</a>
    </div>

    
    <a href="/publication/magellan/"  class="summary-link">
      <div class="article-style">
        Computational models have become essential tools for understanding signalling networks and their non-linear dynamics. However, these …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author, Co-corresponding author"></i>, <span >
      Charlie George Barker</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Yuxin Sun</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Theodoros I. Roumeliotis</span>, <span >
      Jyoti S. Choudhary</span>, <span >
      Jasmin Fisher</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2025.05.16.653408v1" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/jfisher-lab/MAGELLAN" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/auto_drug/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/https://doi.org/10.1101/2025.05.16.653408" target="_blank" rel="noopener">
  DOI
</a>



    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/publication/magellan/" >
        <img src="/publication/magellan/featured_hu466152091706094147.webp" height="75" width="150"
            alt="MAGELLAN: Automated Generation of Interpretable Computational Models for Biological Reasoning" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/publication/nsclc_2025/" >Predicting Personalised Therapeutic Combinations in Non-Small Cell Lung Cancer Using In Silico Modelling</a>
    </div>

    
    <a href="/publication/nsclc_2025/"  class="summary-link">
      <div class="article-style">
        The disease burden from non-small cell lung cancer (NSCLC) adenocarcinoma is substantial, with around a million new cases diagnosed …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Charlie George Barker</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Ashley Nicholls</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Matt P. Handler</span>, <span >
      Lisa Pickard</span>, <span >
      Amna Shah</span>, <span >
      David Walter</span>, <span >
      Etienne De Braekeleer</span>, <span >
      Udai Banerji</span>, <span >
      Jyoti Choudhary</span>, <span >
      Saif Ahmed</span>, <span >
      Ultan McDermott</span>, <span >
      Gregory J.  Hannon</span>, <span >
      Jasmin Fisher</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2025.01.07.631497v1" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/radiotherapy/">
    Project
  </a>
  











    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/publication/nsclc_2025/" >
        <img src="/publication/nsclc_2025/featured_hu17159481087377367445.webp" height="111" width="150"
            alt="Predicting Personalised Therapeutic Combinations in Non-Small Cell Lung Cancer Using In Silico Modelling" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/publication/sae_cooc_2024/" >Compositionality and Ambiguity: Latent Co-occurrence and Interpretable Subspaces</a>
    </div>

    
    <a href="/publication/sae_cooc_2024/"  class="summary-link">
      <div class="article-style">
        Sparse AutoEncoder (SAE) latents show promise as a method for extracting interpretable features from large language models (LM), but …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Hardik Bhatnagar</span>, <span >
      Joseph Bloom</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.lesswrong.com/posts/WNoqEivcCSg8gJe5h/compositionality-and-ambiguity-latent-co-occurrence-and" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/MClarke1991/sae_cooccurrence" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://feature-cooccurrence.streamlit.app/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/sae_cooccurrence/">
    Project
  </a>
  







  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://youtu.be/OyKXjdA09fE?si=gD9IRD0ohVMEbYwr" target="_blank" rel="noopener">
  Video
</a>





    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/publication/sae_cooc_2024/" >
        <img src="/publication/sae_cooc_2024/featured_hu14183490116378626769.webp" height="99" width="150"
            alt="Compositionality and Ambiguity: Latent Co-occurrence and Interpretable Subspaces" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/publication/howell2023/" >Localized immune surveillance of primary melanoma in the skin deciphered through executable modeling</a>
    </div>

    
    <a href="/publication/howell2023/"  class="summary-link">
      <div class="article-style">
        While skin is a site of active immune surveillance, primary melanomas often escape detection. Here, we have developed an in silico …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
      <div>
        

  <span >
      Rowan Howell</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      James Davies</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Anna Appios</span>, <span >
      Inês Mesquita</span>, <span >
      Yashoda Jayal</span>, <span >
      Ben Ringham-Terry</span>, <span >
      Isabel Boned Del Rio</span>, <span >
      Jasmin Fisher</span>, <span >
      Clare L. Bennett</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.science.org/doi/epdf/10.1126/sciadv.add1992" target="_blank" rel="noopener">
  PDF
</a>





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/JFisherLab/Melanoma-LC" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/combination_therapy/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1126/sciadv.add1992" target="_blank" rel="noopener">
  DOI
</a>



    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/publication/howell2023/" >
        <img src="/publication/howell2023/featured_hu11448512100955574606.webp" height="133" width="150"
            alt="Localized immune surveillance of primary melanoma in the skin deciphered through executable modeling" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/publication/talarmain2022/" >HOXA9 has the hallmarks of a biological switch with implications in blood cancers</a>
    </div>

    
    <a href="/publication/talarmain2022/"  class="summary-link">
      <div class="article-style">
        Blood malignancies arise from the dysregulation of haematopoiesis. The type of blood cell and the specific order of oncogenic events …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
      <div>
        

  <span >
      Laure Talarmain</span>, <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      David Shorthouse</span>, <span >
      Lilia Cabrera-Cosme</span>, <span >
      David G. Kent</span>, <span >
      Jasmin Fisher</span>, <span >
      Benjamin A. Hall</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/articles/s41467-022-33189-w.pdf" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.5281/zenodo.6913664" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-022-33189-w/MediaObjects/41467_2022_33189_MOESM5_ESM.xlsx" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/cancer_evolution/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-022-33189-w" target="_blank" rel="noopener">
  DOI
</a>



    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/publication/talarmain2022/" >
        <img src="/publication/talarmain2022/featured_hu799304893979585707.webp" height="82" width="150"
            alt="HOXA9 has the hallmarks of a biological switch with implications in blood cancers" loading="lazy">
      </a>
    
  </div>
</div>

  

  
  
  

    
    
      
    

    
    
    
    
      
    

    

    <div class="see-all">
      <a href="/publication/">
        See all publications
        <i class="fas fa-angle-right"></i>
      </a>
    </div>
  

</div>


  
    </div>
  

  </div>
</section>

  

  
  
    





























































<section id="talks" class="home-section wg-collection  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  ">
    
      
        <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Recent &amp; Upcoming Talks</h1>
          
        </div>
      
    
  

    










  









  
  
  
  
  






















  




<div class="col-12 col-lg-8">

  

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/talk/containing-spillover-with-minimal-supervision/" >Containing Spillover with Minimal Supervision</a>
    </div>

    
    <a href="/talk/containing-spillover-with-minimal-supervision/"  class="summary-link">
      <div class="article-style">
        Winner of 3rd place in the SPAR Symposium 2024 poster competition.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      
      <div>
        <span>
          May 21, 2025
        </span>
        
      </div>
      

      
      <div>
        

  <span >
      Ariana Azarbal</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span class="author-highlighted">
      Matthew A. Clarke</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Jorio Cocola</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Cailley Factor</span><i class="author-notes fas fa-info-circle" data-toggle="tooltip" title="Joint first author"></i>, <span >
      Alex Cloud</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  









  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/gradient_routing/">
    Project
  </a>
  











    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/talk/containing-spillover-with-minimal-supervision/" >
        <img src="/talk/containing-spillover-with-minimal-supervision/featured_hu9468864835111936984.webp" height="54" width="150"
            alt="Containing Spillover with Minimal Supervision" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/talk/features-that-fire-together-wire-together-examining-co-occurence-of-sae-features/" >Features that Fire Together Wire Together: Examining Co-occurence of SAE Features </a>
    </div>

    
    <a href="/talk/features-that-fire-together-wire-together-examining-co-occurence-of-sae-features/"  class="summary-link">
      <div class="article-style">
        Sparse autoencoders (SAE) aim to decompose activation patterns in neural networks into interpretable features. This promises to let us …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      
      <div>
        <span>
          Oct 11, 2024
        </span>
        
      </div>
      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Joseph Bloom</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  









  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/sae_cooccurrence/">
    Project
  </a>
  







  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://youtu.be/OyKXjdA09fE?si=gD9IRD0ohVMEbYwr" target="_blank" rel="noopener">
  Video
</a>





    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/talk/features-that-fire-together-wire-together-examining-co-occurence-of-sae-features/" >
        <img src="/talk/features-that-fire-together-wire-together-examining-co-occurence-of-sae-features/featured_hu15228915930909182962.webp" height="84" width="150"
            alt="Features that Fire Together Wire Together: Examining Co-occurence of SAE Features " loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screening/" >Predicted combination treatments for NSCLC using in silico CRISPR screening</a>
    </div>

    
    <a href="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screening/"  class="summary-link">
      <div class="article-style">
        Lung cancer is the leading cause of cancer mortality world-wide, with over half of lung cancer patients relying on radiotherapy (RT). …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      
      <div>
        <span>
          May 1, 2024
        </span>
        
        <span class="middot-divider"></span>
        <span>Kennedy Lecture Theatre, UCL Great Ormond Street Institute of Child Health</span>
        
      </div>
      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Charles Barker</span>, <span >
      Ashley Nicholls</span>, <span >
      Matt Handler</span>, <span >
      Lisa Pickard</span>, <span >
      Saif Ahmad</span>, <span >
      Udai Banerji</span>, <span >
      Jyoti Choudhary</span>, <span >
      Gregory J. Hannon</span>, <span >
      Jasmin Fisher</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  









  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/combination_therapy/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/cancer_evolution/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/radiotherapy/">
    Project
  </a>
  











    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screening/" >
        <img src="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screening/featured_hu13279725097544183188.webp" height="75" width="150"
            alt="Predicted combination treatments for NSCLC using in silico CRISPR screening" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screens/" >Predicted combination treatments for NSCLC using in silico CRISPR screens</a>
    </div>

    
    <a href="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screens/"  class="summary-link">
      <div class="article-style">
        Lung cancer is the leading cause of cancer mortality world-wide, with over half of lung cancer patients relying on radiotherapy (RT). …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      
      <div>
        <span>
          Feb 27, 2024 &mdash; Feb 28, 2024
        </span>
        
        <span class="middot-divider"></span>
        <span>The Discovery Centre,</span>
        
      </div>
      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span>, <span >
      Charles Barker</span>, <span >
      Ashley Nicholls</span>, <span >
      Matt Handler</span>, <span >
      Lisa Pickard</span>, <span >
      Saif Ahmad</span>, <span >
      Udai Banerji</span>, <span >
      Jyoti Choudhary</span>, <span >
      Gregory J. Hannon</span>, <span >
      Jasmin Fisher</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  









  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/combination_therapy/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/cancer_evolution/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/radiotherapy/">
    Project
  </a>
  











    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screens/" >
        <img src="/talk/predicted-combination-treatments-for-nsclc-using-in-silico-crispr-screens/featured_hu1146593615681793293.webp" height="83" width="150"
            alt="Predicted combination treatments for NSCLC using in silico CRISPR screens" loading="lazy">
      </a>
    
  </div>
</div>

  
    







  







  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/talk/predicting-personalised-treatment-combinations-with-in-silico-crispr-screens/" >Predicting personalised treatment combinations with in silico CRISPR screens</a>
    </div>

    
    <a href="/talk/predicting-personalised-treatment-combinations-with-in-silico-crispr-screens/"  class="summary-link">
      <div class="article-style">
        Targeted therapies can be very effective in the short term, but often are stymied by resistance in the long term. We explore two …
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      
      <div>
        <span>
          Jan 29, 2024 &mdash; Jan 30, 2024
        </span>
        
        <span class="middot-divider"></span>
        <span>The Francis Crick Institute</span>
        
      </div>
      

      
      <div>
        

  <span class="author-highlighted">
      Matthew A. Clarke</span>
      </div>
      
    </div>

    
    <div class="btn-links">
      








  









  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/combination_therapy/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/cancer_evolution/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/radiotherapy/">
    Project
  </a>
  











    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/talk/predicting-personalised-treatment-combinations-with-in-silico-crispr-screens/" >
        <img src="/talk/predicting-personalised-treatment-combinations-with-in-silico-crispr-screens/featured_hu16405687960092544445.webp" height="112" width="150"
            alt="Predicting personalised treatment combinations with in silico CRISPR screens" loading="lazy">
      </a>
    
  </div>
</div>

  

  
  
  

    
    
      
    

    
    
    
    
      
    

    

    <div class="see-all">
      <a href="/event/">
        See all events
        <i class="fas fa-angle-right"></i>
      </a>
    </div>
  

</div>


  
    </div>
  

  </div>
</section>

  

  
  
    





























































<section id="section-tag_cloud" class="home-section wg-tag-cloud  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  ">
    
      
        <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Popular Topics</h1>
          
        </div>
      
    
  

    












  








<div class="col-12 col-lg-8">
  

  

    
    
    
    
    
    

    <div class="tag-cloud">
      
        
        
        
        <a href="/tag/acute-myeloid-leukemia/" style="font-size:0.8rem">Acute Myeloid Leukemia</a>
      
        
        
        
        <a href="/tag/ai-safety/" style="font-size:0.8rem">AI Safety</a>
      
        
        
        
        <a href="/tag/aml/" style="font-size:0.8rem">AML</a>
      
        
        
        
        <a href="/tag/biomodelanalyzer/" style="font-size:1.9570137583148124rem">BioModelAnalyzer</a>
      
        
        
        
        <a href="/tag/bma/" style="font-size:2.462900158640009rem">BMA</a>
      
        
        
        
        <a href="/tag/breast-cancer/" style="font-size:1.7169117098737146rem">Breast Cancer</a>
      
        
        
        
        <a href="/tag/cancer/" style="font-size:2.3405435788154145rem">Cancer</a>
      
        
        
        
        <a href="/tag/combination-therapy/" style="font-size:2.1971158067559102rem">Combination Therapy</a>
      
        
        
        
        <a href="/tag/computational-modelling/" style="font-size:1.9570137583148124rem">Computational Modelling</a>
      
        
        
        
        <a href="/tag/covid-19/" style="font-size:0.8rem">COVID-19</a>
      
        
        
        
        <a href="/tag/ddr/" style="font-size:0.8rem">DDR</a>
      
        
        
        
        <a href="/tag/drug-discovery/" style="font-size:1.0401020484410983rem">Drug Discovery</a>
      
        
        
        
        <a href="/tag/evolution/" style="font-size:1.3785068791574062rem">Evolution</a>
      
        
        
        
        <a href="/tag/heterogeneity/" style="font-size:0.8rem">Heterogeneity</a>
      
        
        
        
        <a href="/tag/llm/" style="font-size:0.8rem">LLM</a>
      
        
        
        
        <a href="/tag/lung-cancer/" style="font-size:1.2263396659274755rem">Lung Cancer</a>
      
        
        
        
        <a href="/tag/machine-learning/" style="font-size:0.8rem">Machine Learning</a>
      
        
        
        
        <a href="/tag/network-modelling/" style="font-size:1.5071624247730138rem">Network Modelling</a>
      
        
        
        
        <a href="/tag/nsclc/" style="font-size:1.5071624247730138rem">NSCLC</a>
      
        
        
        
        <a href="/tag/radiotherapy/" style="font-size:1.8843932794826028rem">Radiotherapy</a>
      
    </div>
  

</div>


  
    </div>
  

  </div>
</section>

  

  
  
    





























































<section id="contact" class="home-section wg-contact  "  >
 <div class="home-section-bg " >
   
 </div>
  <div class="container">

  
    <div class="row  ">
    
      
        <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Contact</h1>
          
        </div>
      
    
  

    






















<div class="col-12 col-lg-8">
  

  

    
    
      
      
      
    

    <div class="mb-3">
      <form name="contact" method="POST" action="https://formspree.io/f/mdovzprn"   >
        <div class="form-group form-inline">
          <label class="sr-only" for="inputName">Name</label>
          <input type="text" name="name" class="form-control w-100" id="inputName" placeholder="Name" required>
        </div>
        <div class="form-group form-inline">
          <label class="sr-only" for="inputEmail">Email</label>
          <input type="email" name="email" class="form-control w-100" id="inputEmail" placeholder="Email" required>
        </div>
        <div class="form-group">
          <label class="sr-only" for="inputMessage">Message</label>
          <textarea name="message" class="form-control" id="inputMessage" rows="5" placeholder="Message" required></textarea>
        </div>
        
        <div class="d-none">
          <label>Do not fill this field unless you are a bot: <input name="_gotcha"></label>
        </div>
        
        <button type="submit" class="btn btn-primary px-3 py-2 w-100">Send</button>
      </form>
    </div>
  

  <ul class="fa-ul">

    

    

    
    
      
      <li>
        <i class="fa-li fas fa-map-marker fa-2x" aria-hidden="true"></i>
        <span id="person-address">UK AI Security Institute, London, </span>
      </li>
    

    

    

    

    
    

  </ul>

  

</div>


  
    </div>
  

  </div>
</section>

  


  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  












  
  
  
  
  













  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    © 2025 Matthew A. Clarke. This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0" rel="noopener noreferrer" target="_blank">CC BY NC ND 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-nd/4.0" rel="noopener noreferrer" target="_blank" aria-label="Creative Commons">
      <i class="fab fa-creative-commons fa-2x" aria-hidden="true"></i>
      <i class="fab fa-creative-commons-by fa-2x" aria-hidden="true"></i>
      
        <i class="fab fa-creative-commons-nc fa-2x" aria-hidden="true"></i>
      
      
        <i class="fab fa-creative-commons-nd fa-2x" aria-hidden="true"></i>
      
    </a>
  </p>





  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://hugoblox.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Hugo Blox Builder</a> — the free, <a href="https://github.com/HugoBlox/hugo-blox-builder" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  


<script src="/js/vendor-bundle.min.50933d940896e49f984a778650d5f7f5.js"></script>




  
    <script src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js" integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin="anonymous"></script>
  

  
  

  













  
  <script id="search-hit-fuse-template" type="text/x-template">
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script>
  
    <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
  












  
  
  
  
  
  
  

















<script id="page-data" type="application/json">{"use_headroom":false}</script>











  
  


<script src="/en/js/wowchemy.min.7f5ebaff62ae468cff8bb3dd1337bb9b.js"></script>







  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        
        <pre><code></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>


  <script src="/js/wowchemy-publication.9c0e895144aef5a693008b5c5d450147.js" type="module"></script>


















</body>
</html>
